src/rtcModule/webrtc.cpp

```diff
@@ -2192,11 +2328,18 @@ Session::Session(Call& call, RtMessage& packet, SdpKey sdpkey)
     mName = "sess[" + mSid.toString() + "]";
     auto wptr = weakHandle();
     mSetupTimer = setTimeout([wptr, this] {
-        if (wptr.deleted())
+        if (wptr.deleted() || mState >= kStateInProgress)
+        {
```

Comment:
In case the state is in-progress or higher, shouldn't this timer have been stopped already?

Reply:
In actual code state, this timer is not stopped. we can change for that behaviour